docs/resources/bypassrls.md

```diff
@@ -1,6 +1,6 @@
 ---
 # generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "pgrole_bypassrls Resource - pgrole"
+page_title: "pgrole_bypassrls Resource - terraform-provider-pgrole"
```

Comment:
this was generated by running `make generate`, not sure why i have different output :thinking: